marklogic-data-hub/src/test/java/com/marklogic/hub/DataHubInstallTest.java

```diff
@@ -21,12 +21,19 @@
 import static org.junit.Assert.assertTrue;
 
 public class DataHubInstallTest extends HubTestBase {
+    private static int afterTelemetryInstallCount = 0;
```

Comment:
A bunch of our tests call `installHub`. What if one of the other test classes runs before this one? Wouldn't it be better to check `getTelemtryInstallCount` before and after the install and look for the +1?

Reply:
There's no +1. It only stores the count until it transmits it, then dumps it, usually within 30 seconds. I'm more concerned the check won't pass if a machine is a bit slow at running the tests - as it'll show null instead of 1.